lib/app/utils/measure_operations.rb

```diff
@@ -41,5 +41,64 @@ def async_submit_data(params_resource)
       }
       LoggedRestClient.post(@instance.url + '/$import', params_resource.to_json, headers)
     end
+
+    def get_required_library_ids(library)
```

Comment:
These functions are super clean!

NIT: Only suggestion would be to re-order them based on the order in which they are called, so:

1. get_all_dependent_valuesets
2. get_measure_resource
3. get_library_resource
4. get_all_library_dependent_valuesets
5. get_required_library_ids
6. get_library_resource
7. get_valueset_urls